### PR TITLE
[BALANCE] Reduces Weight Size of all Gas Masks + Changelog

### PR DESCRIPTION
**Pull request by Steelpoint**
All gas masks are now a W_Class 2 item, this includes the syndicate voice changer.

This also includes a changelog for this change.

Linking to prior PR: https://github.com/tgstation/-tg-station/pull/10384

### CODE REVIEW

File: html/changelogs/Steelpoint-PR-#10686.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "All Gas Masks, this includes SWAT Masks and Syndicate Voice Changers, are now a weight two item, meaning they can fit inside pockets or boxes for example."
```
Comment (Jordie0608) [inline html/changelogs/Steelpoint-PR-#10686.yml]: You'll want to remove this if you're making an entry here. https://github.com/tgstation/-tg-station/blob/master/html/changelogs/.all_changelog.yml#L1257

Comment (Steelpoint) [inline html/changelogs/Steelpoint-PR-#10686.yml]: I forgot I did that one.

I'll just remove the SWAT Mask entry.

Comment (Jordie0608) [inline html/changelogs/Steelpoint-PR-#10686.yml]: You didn't; I made it when I updated the changelog a few hours ago.

Comment (Steelpoint) [inline html/changelogs/Steelpoint-PR-#10686.yml]: Neato.

**Comment by Steelpoint** [comment]
This PR was put up due to discussion held on the #coderbus IRC channel and the argument was made that if I was to change the SWAT gas masks to a lower weight type that I should act consistently and change all gas masks to a lower weight type.

**Comment by Ikarrus** [comment]
I still feel this is entirely unnecessary. 

Why do we even have size oimits if we're going to change them because they's slightly inconvenient?

**Comment by Steelpoint** [comment]
You can already fit a breath mask inside a internals box, as well as a lot of other larger items.

The difference between a breath mask and a gas mask is only small, and the differences are usually more of a edge case than one that is game breaking.

**Comment by Ikarrus** [comment]
That's the thing, I don't want to remove the little things that make them different.

Gas masks protect your face and hide it from view as well, in exchange for being slightly more cumbersome.

Because it (should) slightly inconvenience you isn't a good enough reason to change this.

**Comment by MrStonedOne** [comment]
What is with this push away from things that make the game have any challenge or difficulty or inconvenience?

For example: People buffed mining spawn rates to the point that science will never have to decide how resources are spent, even if there is only one miner, by 20 minutes in they will have more then they will need as long as they aggressively upgrade things. That's like playing fast money maps in starcraft to remove the resource wars mini game. Like I get the appeal, you want to remove all but the core challenge, I used to do it two, but that challenge adds depth to the fun as long as you can get away from hating it long enough to go with the flow and enjoy it.

It's the same here.

**Comment by Cheridan** [comment]
convenience creep

**Comment by Steelpoint** [comment]
No skin off my back, this was done out of popular request from the Irc.

**Pull request closed without merging**